lc-hax/Scripts/Modules/Possession/Controllers/MaskedController.cs

```diff
@@ -1,9 +1,23 @@
+using GameNetcodeStuff;
+
 class MaskedPlayerController : IEnemyController<MaskedPlayerEnemy> {
+    void SetstartingKillAnimationLocalClient(MaskedPlayerEnemy enemy, bool value) =>
+        enemy.Reflect().SetInternalField("startingKillAnimationLocalClient", value);
+
+    bool GetinKillAnimation(MaskedPlayerEnemy enemy) => enemy.Reflect().GetInternalField<bool>("inKillAnimation");
+
     public void UsePrimarySkill(MaskedPlayerEnemy enemy) => enemy.SetHandsOutServerRpc(!enemy.creatureAnimator.GetBool("HandsOut"));
 
     public void UseSecondarySkill(MaskedPlayerEnemy enemy) => enemy.SetCrouchingServerRpc(!enemy.creatureAnimator.GetBool("Crouching"));
 
     public float InteractRange(MaskedPlayerEnemy _) => 1.0f;
 
     public bool SyncAnimationSpeedEnabled(MaskedPlayerEnemy _) => false;
+
+    public void OnOutsideStatusChange(MaskedPlayerEnemy enemy) => enemy.StopSearch(enemy.searchForPlayers, true);
+
+
+    public void OnCollideWithPlayer(MaskedPlayerEnemy enemy, PlayerControllerB player) {
+
+    }
```

Comment:
Will this be populated?

Reply:
No, we can remove it, i might remove it entirely and patch the check so the normal enemy collision would work